README.md

```diff
@@ -23,7 +23,7 @@
   </a>
 </p>
 
-Turbo is an next-generation toolchain for frontend development, written in Rust. It consists of 3 major parts:
+Turbo is a next-generation, Blazingly Fast™, toolchain for frontend development, written in Rust. It consists of 3 major parts:
```

Comment:
Cc @Rich-Harris, _pulitzer prize-winning former purveyor of fake news at the NY Times, and author of another not angular framework_ 

Should it be Blazing Fast or Blazingly Fast? Asking so I can file the trademark.

> Blazing could be used to intensify fast, just as something can be burning hot or sopping wet, which sound redundant but are idiomatic. Blazingly, the usual adverbial form, would of course work in this manner. We could alternatively read blazing and fast as parallel descriptors for speed, just as we can have soothing sweet tea or an amazing technicolor dreamcoat. https://english.stackexchange.com/a/139321

Reply:
Interested to see Rich's thoughts on this

Reply:
I'll merge the change!

Reply:
"Author of another not angular framework" definitely worthy to be Rich's new Twitter bio